### PR TITLE
Get rid py27

### PR DESCRIPTION
**Pull request by menshikh-iv**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

It's time ....

Partial fix for https://github.com/RaRe-Technologies/gensim/issues/2548

Missing:
- Understand and fix python versions (3.5 instead of 2.7)

### CODE REVIEW

**Comment by menshikh-iv** [comment]
@conda-forge-admin, please rerender

**Comment by menshikh-iv** [comment]
@conda-forge-admin, please rerender

**Comment by menshikh-iv** [comment]
@conda-forge-admin, please rerender

**Comment by menshikh-iv** [comment]
@conda-forge-admin, please rerender

**Comment by menshikh-iv** [comment]
hey @conda-forge/core, can you help us with this issue (see comment above)?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   skip: true  # [win32 or osx]
```
Comment (jjhelmus) [inline recipe/meta.yaml]: Adding `skip: true  # [py==27]` will skip the Python 2.7 builds on all platforms.  This is likely easier than modifying the `conda-forge.yml`file.

**Comment by menshikh-iv** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (menshikh-iv) [reply recipe/meta.yaml]: Doesn't work, still can't re-render :(

**Comment by jjhelmus** [comment]
@menshikh-iv The bot is having issues with re-rendering.  I've done it locally and submitted a PR with the change in #38

**Comment by menshikh-iv** [comment]
@jjhelmus awesome, thank for your help


continued in #38

**Pull request closed without merging**